src/libultra/os/resetglobalintmask.c

```diff
@@ -1,3 +1,8 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/resetglobalintmask/__osResetGlobalIntMask.s")
+void __osResetGlobalIntMask(u32 mask) {
+    register s32 prevInt = __osDisableInt();
+
+    __OSGlobalIntMask &= ~(mask & ~0x401);
```

Comment:
```suggestion
    __OSGlobalIntMask &= ~(mask & ~(0x400 | 0x1));
```
While I don't understand the code, and I know boot has different standards than regular code. But if this is a bit-wise operation, then I like to make the bits explicit